src/scripts/utils/swRegister.es6

```diff
@@ -0,0 +1,5 @@
+if ('serviceWorker' in navigator) {
+	navigator.serviceWorker.register('serviceWorker.js').then((registration) => {
+		console.log(registration)
+	})
```

Comment:
Se SW jsem teprve začínal, tak jsem tohle moc neřešil. K čemu ti vlastně pomůže zalogování chyby? V konzoli je to houby platné. Postupně vzniká snaha dostat na front-end Sentry.